### PR TITLE
TR feedback by moio

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/9512
Broken PR, to be closed...

### CODE REVIEW

File: modules/installation/pages/install-ibmz.adoc [hunk 1]
```diff
@@ -90,6 +90,8 @@ The hostname should contain less than eight characters.
 
 This section covers the installation of {productname}{nbsp}{productnumber} as an extension to {sls}{nbsp}{sles-version}{nbsp}{sp-version}.
 
+// TBD: Is this correct? SUSE Manager as an extension? Doesn't SLES15SP1 on z also have the new module concept with SUSE Manager being a base product?
```
Comment (juliogonzalez) [inline modules/installation/pages/install-ibmz.adoc]: For s390x it's also a base product AFAIK, as the producer documented at https://github.com/SUSE/spacewalk/wiki/SCC-QA-proxy#scratch40 doesn't say anything about adding it as an extension.

@Bischoff can you confirm that you tested s390x as a regular base product, installing from unified installer?

**Comment by keichwa** [comment]
> @keichwa can you please generate a PDF? Reviewing file by file is hard, with 18 failed changed :-)

Stupid me, I did a 'git rebase master' - most changes are completely unrelated.  I guess I must create a new PR... I'm only interested in:

- modules/administration/pages/maintenance-window.adoc 
- modules/upgrade/pages/migrate-4x-4x.adoc 
- modules/upgrade/pages/server-update.adoc

**Comment by Loquacity** [comment]
> Stupid me, I did a 'git rebase master' - most changes are completely unrelated. I guess I must create a new PR... I'm only interested in:
> 

:flushed:

**Comment by juliogonzalez** [comment]
No problem, just close this one, open the new one, and will review there :-)

**Comment by keichwa** [comment]
> No problem, just close this one, open the new one, and will review there :-)

Ok, done. For your s390x/ibm z question, we probably create a new issue...

**Pull request closed without merging**